src/pyrokinetics/gk_code/gs2.py

```diff
@@ -617,6 +617,21 @@ def _detect_normalisation(self):
             temperatures.append(temp)
             masses.append(mass)
 
+        adiabatic_electron_flags = ["iphi00=2", "field-line-average-term"]
+
+        if (
+            not found_electron
+            and self.data["dist_fn_knobs"]["adiabatic_option"]
+            in adiabatic_electron_flags
+        ):
+            found_electron = True
+            nspec = self.data["species_knobs"]["nspec"] + 1
+            species_key = f"species_parameters_{nspec}"
```

Comment:
Just for parity with the earlier loop

```suggestion
            isp = self.data["species_knobs"]["nspec"] + 1
            species_key = f"species_parameters_{isp + 1}"
```

Reply:
This is now no longer relevant